packages/docusaurus/lib/server/load/routes.js

```diff
@@ -68,17 +76,23 @@ async function loadRoutes(pluginsRouteConfigs) {
       throw new Error(`path: ${routePath} need a component`);
     }
     const componentPath = getModulePath(component);
-    addRoutesAsyncModule(routePath, componentPath);
 
-    const genImportStr = (modulePath, prefix, name) => {
-      const chunkName = genChunkName(name || modulePath, prefix);
+    const genImportChunk = (modulePath, prefix, name) => {
+      const chunkName = genChunkName(name + modulePath || '', prefix);
```

Comment:
I had to change this as there were naming collisions for the chunks generated for `/blog`. There should be a better way to generate unique chunks for this.